lib/src/editor.dart

```diff
@@ -959,19 +958,6 @@ pre {
   Future<void> _onContentSizeChangedOnIOS(
     InAppWebViewController controller,
     Size oldContentSize,
-    Size newContentSize
-  ) async {
-    if (_isFocusing) {
-      return;
-    }
-
-    final contentHeight = max(oldContentSize.height, newContentSize.height);
-    log('HtmlEditorState::_onContentSizeChanged:contentHeight: $contentHeight');
-    final documentHeight = _documentHeight ?? 0;
-    if (contentHeight > documentHeight && mounted) {
-      setState(() {
-        _documentHeight = contentHeight + _offsetHeight;
-      });
-    }
-  }
+    Size newContentSize,
+  ) => onDocumentChanged();
```

Comment:
It may affect composer content editing. Please make sure to check again.

Reply:
No issue found for composer

Reply:
Please test the `lazy load image` case with `Forward/Reply` emails. Because we have fixed that bug before.

Reply:
Tested and confirmed both `lazy load image` and `rich text`

Reply:
Please attach demo on Tmail project side

Reply:
Done